stages/test/test_zipl_inst.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
+
+import os
+import textwrap
+
+from unittest.mock import patch
+
+from osbuild.testutil import make_fake_tree
+
+STAGE_NAME = "org.osbuild.zipl.inst"
+
+
+@patch("subprocess.run")
+def test_zipl_inst_default(mocked_run, tmp_path, stage_module):
+    make_fake_tree(tmp_path, {
+        "/boot/loader/entries/ostree-1-fedora-coreos.conf": textwrap.dedent("""\
+        title Fedora CoreOS 39.20240206.dev.0 (ostree:0)
```

Comment:
Thank you for adding meaningful values in here - I really like it when tests can be read like documentation/hints for how the "real world" will look like!